Kenneth-Macharia/fastfoodfast_flaskAPI
# #160905488 various bug fixes

What does this PR do?
Implements various bug fixes to code

Description of Task to be completed?
-Fixes attempt to amend an order status without a key-value pair input.
-Refactoring of tests after implementing the above code changes.
-Code polishing

How should this be manually tested?
-See the project docs for app setup & run instructions

Any background context you want to provide?
-Changes recommended by LFA

What are the relevant pivotal tracker stories?
#160905488
